### PR TITLE
Zulia Version Information

### PR DESCRIPTION
**Pull request by kdallasd**
Adds Zulia version information to the command-line, REST service, Stats document, and attached nodes.

### CODE REVIEW

File: zulia-common/src/main/java/io/zulia/util/ZuliaVersion.java [hunk 1]
```diff
@@ -24,4 +24,11 @@ public static String getVersion() {
 		return VERSION.version;
 	}
 
+	/**
+	 * Gets the version of Zulia when versioning was added
+	 * @return the version of Zulia when versioning was added
+	 */
+	public static String getVersionAdded() {
+		return "2.1.1";
```
Comment (mdavis95) [inline zulia-common/src/main/java/io/zulia/util/ZuliaVersion.java:32]: Make this should be 2.0.2 or 2.1.0

File: zulia-server/src/main/java/io/zulia/server/config/cluster/MongoNodeService.java [hunk 2]
```diff
@@ -82,6 +84,11 @@ public void updateHeartbeat(String serverAddress, int servicePort) {
 
 	}
 
+	@Override
+	public void updateVersion(String version) {
+		getCollection().updateMany(new Document(), new Document("version", ZuliaVersion.getVersion()));
```
Comment (mdavis95) [inline zulia-server/src/main/java/io/zulia/server/config/cluster/MongoNodeService.java:89]: This actually should be like updateHeartbeat and take a serverAddress and servicePort so it only updates this node's version

**Review by mdavis95 — CHANGES_REQUESTED** [review]

File: zulia-common/src/main/java/io/zulia/util/ZuliaVersion.java (hunk 1 above)
Comment (kdallasd) [reply zulia-common/src/main/java/io/zulia/util/ZuliaVersion.java]: Updated to 2.1.0 in latest commit

File: zulia-server/src/main/java/io/zulia/server/config/cluster/MongoNodeService.java (hunk 2 above)
Comment (kdallasd) [reply zulia-server/src/main/java/io/zulia/server/config/cluster/MongoNodeService.java]: Added.

**Pull request merged**